### DISCUSSION TITLE
Any plans for implementing more Keyword features?

### DISCUSSION THREAD

**Discussion by GD-Dal** [immich-app/immich#5016] [General]
This app has come to my attention, and it looks very interesting!

But keywords are a big deal for me.
Example:
If I click on information on a picture, all keywords found in the metadata should be displayed. Then, when clicking a keyword, all pictures containing that keyword should be displayed.
Also a menu entry for keywords should be added, so I can get a list of all available keywords, and find pictures simply by clicking a keyword.
Piwigo has things like this implemented if you need a reference

Any plans for implementing a similar feature in Immich?

Thanks

**Comment by jrasm91** [comment]
I think this is related to a bigger upgrade of search/explore which is currently dependent on a decision about a potential pgvector migration (instead of using typesense).
